examples/test/Menu_test.js

```diff
@@ -0,0 +1,158 @@
+    const measureCell = getMeasureCell(0);
+    const menu = getMenu(measureCell);
+
+    await t.hover(measureCell);
+    await t.expect(menu.visible).eql(true);
+
+    const anotherCell = getMeasureGroupCell(0);
+    await t.hover(anotherCell);
+    await t.expect(menu.visible).eql(false);
+});
+
+test('should add totals for first measure, when selected from menu in measure', async (t) => {
+    const measureCell = getMeasureCell(0);
+
+    await t.expect(getPivotTableFooterCell(0, 0).exists).eql(false);
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+
+    await t.expect(getPivotTableFooterCell(0, 0).exists).eql(true);
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(1, 0).exists).eql(false);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.empty);
+});
+
+test('should add totals for all measures, when selected from menu in measure group', async (t) => {
+    const measureCell = getMeasureGroupCell(0);
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.sum[4]);
+});
+
+test('should add totals for one measure and then turn it off', async (t) => {
+    const measureCell = getMeasureCell(0);
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.empty);
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).exists).eql(false);
+});
+
+test('should add totals for all measures and then turn them off', async (t) => {
+    const measureCell = getMeasureGroupCell(0);
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).textContent).eql(totalValues.sum[0]);
+    await t.expect(getPivotTableFooterCell(0, 3).textContent).eql(totalValues.sum[3]);
+    await t.expect(getPivotTableFooterCell(0, 4).textContent).eql(totalValues.sum[4]);
+
+    await clickOnMenuAggregationItem(t, measureCell, '.s-menu-aggregation-sum');
+    await t.expect(getPivotTableFooterCell(0, 0).exists).eql(false);
+});
+
+test('should turn on/off multiple totals', async (t) => {
+    const measureCell = getMeasureCell(0);
+    const measureGroup = getMeasureGroupCell(0);
+
+
```

Comment:
I'm not sure consecutive whitelines will pass `validate`.

Reply:
they pass, but good catch, updated